scripts/tools/tools-schema.json

```diff
@@ -50,7 +50,7 @@
             "properties": {
                 "language": {
                     "type": "string",
-                    "description": "Provide information on how other can integrate with the tool in case it is possible. If your code generator generates Python code, then specify Python, even if code generator itself is written in JavaScript. If we forgot about some language then please add it through a pull request to AsyncAPI website repository.",
+                    "description": "Refers to the language that users can use in the 'runtime'. That is, for generators it should be the language that the generator generates, not the language the generator is written in, itself. But, for example, for Bundler library, even though it operates on YAML and JSON files, more important for users is the fact that the library is written in TypeScript as this information tells them if they can integrate this library into their 'runtime'. In other words, if users have TypeScript codebase and they would like to have a bundling mechanism, they need to know if they can integrate Bundler library into their TypeScript codebase. This property provides information if a user can use given library/script/API specifically in his code. If we forgot about some language then please add it through a pull request to AsyncAPI's 'website' repository.",
```

Comment:
```suggestion
                    "description": "The language referred to in the 'runtime' is the language used by the user, not the language the generator or library is written in. For example, for the Bundler library, users need to know if it can be integrated into their TypeScript codebase. This information determines if a library can be used in a specific codebase. If a language is omitted, it can be added through a pull request to the AsyncAPI website repository.",
```

how about this? it was too repetitive and causing confusion... 

lemme know how you feel about this take on it @akshatnema @aeworxet

Reply:
How about minor change?


**The language referred to is the language used by the user in the 'runtime'**, not the language the generator or library is written in. For example, for the Bundler library, users need to know if it can be integrated into their TypeScript codebase. This information determines if a library can be used in a specific codebase. If a language is omitted, it can be added through a pull request to the AsyncAPI website repository.


But if [Generator](https://github.com/asyncapi/generator) written in JavaScript, generates Python code, how can it be integrated into Python codebase?

Reply:
depends what we understand as runtime. If you have a template that uses JS to specify what will be generated, but the result of the generation in Python app, then the app in runtime is using Python

Reply:
very accurate!

Reply:
@akshatnema 
The description cannot be made shorter then, otherwise it will lose any meaning.

@alequetzalli 
Is this text acceptable? 'Python', 'TypeScript' and 'website' are enclosed in quotes because they are technical terms - so users can be sure they can copy and paste text between the quotes and it will be found in search.

Reply:
1. remove the quotes, they are unnecessary
2. great! I made fixes to your preferred text :) 

---
The language referred to is the language used by the user in the runtime, not the language the generator or library it's written in. For example, the Generator written in JavaScript generates Python code from the JavaScript template and the result of generation is a Python app, so the language for Generator is specified as Python. But for the Bundler library, users need to know if it can be integrated into their TypeScript codebase, so its language is specified as TypeScript. If some language in the schema's enum is omitted, it can be added through a pull request to the AsyncAPI website repository.

Reply:
Can "are" instead of "it's" be specified in this text?

> not the language the generator or library **are** written in

Because language in this case relates to "generator or library", not the "runtime" - just to make it clearer for the user.

Reply:
no, `are` doesn't work in that sentence unless you re-write it

Reply:
Can "is" be left in place then?

> not the language the generator or library **is** written in

Reply:
How about this... 
---

The language referred to is the runtime language selected by the user, not the generator or library language.

Reply:
> The language referred to is the runtime language selected by the user, not the generator or library language.

sounds the best, no `are` and no `it's` no doubts 😄

Reply:
Updated the PR with @alequetzalli's version.